vms/platformvm/service_test.go

```diff
@@ -365,6 +366,11 @@ func TestGetBalance(t *testing.T) {
 	require := require.New(t)
 	service, _, _ := defaultService(t)
 
+	var (
+		feeStaticCalc   = fee.NewStaticCalculator(service.vm.Config.StaticFeeConfig, service.vm.Config.UpgradeConfig)
```

Comment:
nit: this name reads awkwardly to me, rest of PR uses `feeCalculator` which I think is better

Reply:
Done